### ISSUE TITLE
Collecting multiple perspectives: publish a medium post and make a reddit post in r/ethereum,  r/science other similar threads

### ISSUE THREAD

**Issue by himalayajung** [open-science-org/URI#5]
I feel that we should gather the views of scientific researchers and cryptography experts on Unique Researcher Identity (URI). I propose publishing a medium post and making a reddit post in r/ethereum, r/science and other similar threads. 

**Option 1:**
Write a medium post and then create a reddit post linking the medium article.

**Option 2:**
Make a reddit post and medium post together (link them) and then publish a refined medium article incorporating the views of the researchers and experts. Possibly write a white paper on it.

**Comment by abinashk** [comment]
My 👍  on Option 2.

**Comment by oso-network** [comment]
@KeithSSmith Since you have worked more on this concept, do you want to quickly draft the skeleton of a paper on URI and then we will work on it? We can then use parts of the initial version of the paper for the medium article and the reddit post.  We can publish a version 2 after incorporating the feedback from researchers and cryptography experts.

**Comment by KeithSSmith** [comment]
@oso-network Yes, I will work on an initial draft that can be utilized as a skeleton for a white paper/blog and then look for feedback.  Will share the Google Doc with everyone once I have made some headway.

**Comment by himalayajung** [comment]
One perspective: Biometric Identification. Inspired from 'Potential of Biometric Identification for Health' from http://www.intellectualventures.com/ https://www.youtube.com/watch?v=1mHJ2fiXODU&feature=youtu.be
Biometric Identification might be an important piece of the Unique Researcher Identity (URI) or any digital identity concept. However, using biometric identification for URI at the very start might be an overkill.

**Comment by KeithSSmith** [comment]
Medium URI post can be found here: [Medium URI](https://medium.com/@smithkeith/open-science-organization-introduces-the-unique-research-identity-883e182674b8)

For Reddit we should link to this on r/Ethereum and maybe even r/CryptoCurrency

Not sure on the status of a subreddit but we could post it there as well.

After we get feedback we will do another blog with follow up.
